sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -399,6 +399,7 @@ public partial class DiagnosticsOptions
         public int LoggedContentSizeLimit { get { throw null; } set { } }
         public System.Collections.Generic.IList<string> LoggedHeaderNames { get { throw null; } }
         public System.Collections.Generic.IList<string> LoggedQueryParameters { get { throw null; } }
+        public bool SuppressNestedClientSpans { get { throw null; } set { } }
```

Comment:
Do we want to allow customers to suppress inner scopes?

Reply:
discussed offline, agreed to allow users to configure it.